test/routes/github/issues.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { initServer } from '../../../src/server';
+import { getConfig } from '../../../src/config';
+import { IssuePayload } from '../../../src/schemas/github/issue-payload';
+
+describe('POST /github', () => {
+	let payload: IssuePayload;
+
+	beforeEach(() => {
+		payload = {
+			action: 'assigned',
+			assignee: { login: 'SantiMA10' },
+			sender: { login: 'SantiMA10' },
+			repository: {
+				html_url: 'https://github.com/streamdevs/webhook',
+				full_name: 'streamdevs/webhook',
+			},
+		};
+	});
+
+	it('returns a 200 OK', async () => {
+		const subject = await initServer(getConfig());
```

Comment:
We're not setting up the config in the tests.
We should be using an explicit mock configuration instead of knowing that the `getConfig` function internally changes depending on `NODE_ENV`.